src/apis/Dimensions.md

```diff
@@ -6,28 +6,24 @@ wip: true
 
 ```reason
 type displayMetrics = {
-  .
-  "width": float,
-  "height": float,
-  "scale": float,
-  "fontScale": float,
+  width: float,
+  height: float,
+  scale: float,
+  fontScale: float,
 };
 
 type handler = {
-  .
-  "screen": displayMetrics,
-  "window": displayMetrics,
+  screen: displayMetrics,
+  window: displayMetrics,
 };
 
 [@bs.module "react-native"] [@bs.scope "Dimensions"]
 external get: ([@bs.string] [ | `window | `screen]) => displayMetrics = "get";
 [@bs.module "react-native"] [@bs.scope "Dimensions"]
-external addEventListener: ([@bs.string] [ | `change], handler => unit) => unit =
-  "";
+external addEventListener: ([@bs.string] [ | `change], handler => unit) => unit;
 [@bs.module "react-native"] [@bs.scope "Dimensions"]
 external removeEventListener:
-  ([@bs.string] [ | `change], handler => unit) => unit =
-  "";
+  ([@bs.string] [ | `change], handler => unit) => unit;
```

Comment:
While reason 3.6.0 allows omission of `""`, name of the external is explicitly specified elsewhere, so please add `"addEventListener"` and `"removeEventListener"` to the external definitions.

It is already the case in `Dimension.re`, I suppose this file was not in sync with that.